tests/php/Configuration/Parser/MenuParserTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Tests\Configuration\Parser;
+
+use Bolt\Configuration\Parser\MenuParser;
+use PHPUnit\Framework\TestCase;
+use Tightenco\Collect\Support\Collection;
+
+class MenuParserTest extends TestCase
+{
+    /** @var MenuParser */
+    private $menuParser;
+
+    protected function setUp(): void
+    {
+        $this->menuParser = new MenuParser();
+    }
+
+    public function testCanParse(): void
+    {
+        $config = $this->menuParser->parse();
+
+        $this->assertInstanceOf(Collection::class, $config);
```

Comment:
this doesn't need to be asserted when we have strict typing

Reply:
It breaks when menu.yml is invalid.

Reply:
how it breaks? throws an exception? can you test it?

Reply:
Added a @todo for this.  See #379